645b215a29f36ff7921f0180d9477a8a15df88b4
apollo-common/src/main/java/com/ctrip/framework/apollo/common/utils/WebUtils.java
--- 
+++ 
@@ -1,41 +1,41 @@
 /*
- * Copyright 2022 Apollo Authors
+ * Copyright 2023 Apollo Authors
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
  * http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
  */
 package com.ctrip.framework.apollo.common.utils;
 
 import com.google.common.base.Splitter;
 import com.google.common.base.Strings;
 import javax.servlet.http.HttpServletRequest;
 
 /**
  * @author kl (http://kailing.pub)
  * @since 2022/8/12
  */
 public final class WebUtils {
 
   private static final Splitter X_FORWARDED_FOR_SPLITTER = Splitter.on(",").omitEmptyStrings()
       .trimResults();
 
   private WebUtils(){}
 
   public static String tryToGetClientIp(HttpServletRequest request) {
     String forwardedFor = request.getHeader("X-FORWARDED-FOR");
     if (!Strings.isNullOrEmpty(forwardedFor)) {
       return X_FORWARDED_FOR_SPLITTER.splitToList(forwardedFor).get(0);
     }
     return request.getRemoteAddr();
   }
 }